<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tf2RconWindow</class>
 <widget class="QMainWindow" name="Tf2RconWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tf2Rcon</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icon.png</normaloff>:/icons/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0,0,0,0,0,0">
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="kickAll">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Kick all</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="changelevel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Change level</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" rowspan="4">
     <widget class="QListView" name="maps">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="mapFilter">
      <property name="placeholderText">
       <string>Filter maps</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="kickSelected">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Kick</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Connect string:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="2">
     <widget class="QLineEdit" name="sourceTvConnectString">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="exec">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Execute config...</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="2">
     <widget class="QLineEdit" name="connectString">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="command">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Command...</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>SourceTV string:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="5">
     <widget class="QListView" name="users">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="ServerStatusWidget" name="serverStatus" native="true"/>
    </item>
   </layout>
   <zorder>maps</zorder>
   <zorder>changelevel</zorder>
   <zorder>users</zorder>
   <zorder>kickSelected</zorder>
   <zorder>command</zorder>
   <zorder>exec</zorder>
   <zorder>kickAll</zorder>
   <zorder>connectString</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>sourceTvConnectString</zorder>
   <zorder>mapFilter</zorder>
   <zorder>serverStatus</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTf2Rcon">
    <property name="title">
     <string>Tf&amp;2Rcon</string>
    </property>
    <addaction name="connectToServer"/>
    <addaction name="actionSavedServers"/>
   </widget>
   <addaction name="menuTf2Rcon"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="connectToServer">
   <property name="text">
    <string>&amp;Connect to server...</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionSavedServers">
   <property name="text">
    <string>Saved servers...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ServerStatusWidget</class>
   <extends>QWidget</extends>
   <header>serverstatuswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
